lib/model.js

```diff
@@ -1885,6 +1885,35 @@ class Model {
     return this.findOne(options);
   }
 
+  /**
+ * Search for a many instances by its primary key._
+ *
+ * @param  {Array}      param The value of the desired instance's primary key.
+ *
+ * @see
+ * {@link Model.findAll}           for a full explanation of options, Note that options.where is not supported.
+ *
+ * @returns {Promise<Model>}
+ */
+  static findByPks(param) {
+  // return Promise resolved with null if no arguments are passed
+    if ([null, undefined].includes(param)) {
+      return Promise.resolve(null);
+    }
+
```

Comment:
As far as I can see, the code below this line could be refactored to something like:

```javascript
if (!Array.isArray(param)) {
  throw new Error('The argument passed to findByPks must be an array of ids');
}

 return this.findAll({ where: { id: { [Op.in]: param } } });
```

Reply:
@papb please let me know your thoughts on this.

Reply:
Completely agree!